src/ResourceManager/TrafficManager/Commands.TrafficManager2.Test/SessionRecords/Microsoft.Azure.Commands.TrafficManager.Test.ScenarioTests.EndpointTests/TestCreateExistingEndpoint.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
Setting to Always didn't help, I've emailed the DevEx PS team separately for this